query/src/sql/table_option_keys.rs

```diff
@@ -17,15 +17,31 @@ use std::collections::HashSet;
 use lazy_static::lazy_static;
 
 pub const OPT_KEY_DATABASE_ID: &str = "database_id";
+pub const OPT_KEY_SNAPSHOT_LOCATION: &str = "snapshot_loc";
```

Comment:
"snapshot_loc" -> "snapshot_location" is better?

Reply:
If we should keep the backward compatibility, I am afraid the `"snapshot_loc"` have to be kept.

Another option key is being introduced `OPT_KEY_SNAPSHOT_LOCATION = "snapshot_location"`, which will replace the `"snapshot_loc"`. 

engine will try to read `snapshot_location` first, if failed, try to read `snapshot_loc`, thus we can read the existing tables (which have `snapshot_loc` as the location key).

All the new tables will use `"snapshot_location"`